bundle-workflow/src/assemble_workflow/bundle_recorder.py

```diff
@@ -2,23 +2,32 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
+from urllib.parse import urljoin
+
 import yaml
+
 from manifests.bundle_manifest import BundleManifest
```

Comment:
Is it convention to have sorted third party imports followed by sorted internal module imports?

Reply:
Yes, this is output of defaults of `isort`. We can change the convention if we don't like it.

Reply:
personally, I'm not a huge fan of this default but its a nit.

Reply:
Let's consider changing it later!